Emby.Server.Implementations/Library/LibraryManager.cs

```diff
@@ -509,15 +480,15 @@ private Guid GetNewItemIdInternal(string key, Type type, bool forceCaseInsensiti
                 throw new ArgumentNullException(nameof(type));
             }
 
-            if (key.StartsWith(ConfigurationManager.ApplicationPaths.ProgramDataPath, StringComparison.Ordinal))
+            if (key.StartsWith(_configurationManager.ApplicationPaths.ProgramDataPath, StringComparison.Ordinal))
             {
                 // Try to normalize paths located underneath program-data in an attempt to make them more portable
-                key = key.Substring(ConfigurationManager.ApplicationPaths.ProgramDataPath.Length)
+                key = key.Substring(_configurationManager.ApplicationPaths.ProgramDataPath.Length)
                     .TrimStart(new[] { '/', '\\' })
                     .Replace("/", "\\");
```

Comment:
I would actually rather we replace `\` with `/`, as the latter works on both Windows and Linux while the former works only on Windows... but that would most likely break all existing installs :(